src/_locales/de/messages.json

```diff
@@ -39,11 +39,11 @@
         "message": "Deaktiviere das Senden von Daten an Google Analytics",
         "description": "Option title"
     },
-    "optionGmPodcasts": {
```

Comment:
Locales other than English will need to be updated, I decided to update the key names instead of deleting for now since the translations should (in theory) be close to the changed versions.